tests/by-util/test_split.rs

```diff
@@ -145,3 +145,14 @@ fn test_split_str_prefixed_chunks_by_lines() {
     assert_eq!(glob.count(), 10);
     assert_eq!(glob.collate(), at.read(name).into_bytes());
 }
+
+#[test]
+fn test_split_additional_suffix() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let name = "split_additional_suffix";
+    let glob = Glob::new(&at, ".", r"x[[:alpha:]][[:alpha:]].txt$");
+    RandomFile::new(&at, name).add_lines(2000);
+    ucmd.args(&["--additional-suffix", ".txt", name]).succeeds();
```

Comment:
Preface these comments by the fact that I have only rarely used `split`.

The suffix should always be digits (followed by the additional suffix), no?
And I think the destination filename stem needs to be specified as well.

```suggestion
    let glob = Glob::new(&at, ".", r"x[[:digit:]][[:digit:]].txt$");
    RandomFile::new(&at, name).add_lines(2000);
    ucmd.args(&["--additional-suffix", ".txt", name, "x"]).succeeds();
```

Reply:
The default behavior of split are line ending xaa, xab, .... To get numeric ends the flag  d = "use numeric suffixes instead of alphabetic" needs to be used.

Reply:
The suffix is by default alphabetic, e.g. xaa, xab, ...  The behavior described is shown by using the flag
 "d",        "use numeric suffixes instead of alphabetic",
I am not sure, what the "x" is supposed to do in line 155.

Reply:
👍